### DISCUSSION TITLE
/posts/2023/08/29%E4%BD%A0%E6%9D%A5%E4%BD%A0%E4%B9%9F%E5%8F%AF%E4%BB%A5%E5%81%9A%E4%B8%80%E4%B8%AA%E7%BD%91%E7%9B%98%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#25] [Announcements]
# /posts/2023/08/29%E4%BD%A0%E6%9D%A5%E4%BD%A0%E4%B9%9F%E5%8F%AF%E4%BB%A5%E5%81%9A%E4%B8%80%E4%B8%AA%E7%BD%91%E7%9B%98%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/08/29%E4%BD%A0%E6%9D%A5%E4%BD%A0%E4%B9%9F%E5%8F%AF%E4%BB%A5%E5%81%9A%E4%B8%80%E4%B8%AA%E7%BD%91%E7%9B%98%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E

**Comment by iawes** [comment]
考虑开源下代码供小白学习下吗

**Reply by Justin3go** [reply]
暂时没有相关的打算，开源就想把代码写好，毕竟要给大家看，而这就意味着太耗时耗力了

**Comment by huangyfGG** [comment]
请问如何选取要爬取的网站呢，单纯通过搜索引擎+site感觉效果不太好。

**Reply by Justin3go** [reply]
你是指的种子网站吗，如果是这个问题的话：一般来说是一些导航网站，比如有人统计了有关阿里云盘的社区网址做成了一个网站。

> 单纯通过搜索引擎+site感觉效果不太好。

这句话不太理解你要问什么

**Reply by huangyfGG** [reply]
对 种子网站，这个我懂了。那像不同的页面，里面的内容千奇百怪，就比如爬取阿里云链接的话，要怎么建立链接和资源名的对应关系呢，还有验证码。有的可能是链接在前，资源的名字在后面，有的是资源的名字在前面，链接在后面等等。比如 影视资源  http://aliyun.xxxxxx 或者 http://ailiyun/xxxx 影视资源  提取码 xxx

**Reply by Justin3go** [reply]
这部分策略也还有问题
